package-res/reportviewer/reportviewer.js

```diff
@@ -449,13 +449,13 @@ define([ 'common-ui/util/util', 'common-ui/util/timeutil', 'common-ui/util/forma
           }
 
           pc.registerPageNumberChangeCallback(function(pageNumber) {
-            this.pageChanged(this.reportPrompt.panel, pageNumber);
+            this.pageChanged(pageNumber);
           }.bind(this));
         },
 
         pageChanged: function(pageNumber) {
           this._setAcceptedPage('' + (pageNumber - 1));
-          this.reportPrompt.panel.submit({isPageChange: true});
+          this.reportPrompt.panel.submit(this.reportPrompt.panel, {isPageChange: true});
```

Comment:
See https://github.com/pentaho/pentaho-platform-plugin-common-ui/blob/master/package-res/resources/web/prompting/components/SubmitPromptComponent.js#L116
It uses _submit -> and set promptPanel argument to submit function:
https://github.com/pentaho/pentaho-platform-plugin-common-ui/blob/master/package-res/resources/web/prompting/PromptPanel.js#L746
As a result without promptPanel arg in Report Viewer we loose options...